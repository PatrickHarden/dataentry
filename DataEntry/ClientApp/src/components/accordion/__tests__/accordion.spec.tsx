import React from 'react';
import { mount, shallow } from 'enzyme';

import Accordion from '../accordion';

describe('components', () => {

  describe('accordion', () => {

    let props:any;

    beforeEach(() => {
        props = {
            title: 'Accordion Title',
            loadOpen: true,
            alertOn: false,
            titlePrefix: "1"
        }
    });

    // snapshot render
    it('renders according to snapshot', () => {
        expect(shallow(<Accordion {...props} />)).toMatchSnapshot();
    });

  });
});