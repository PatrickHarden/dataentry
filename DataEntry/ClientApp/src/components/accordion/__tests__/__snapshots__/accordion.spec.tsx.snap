// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components accordion renders according to snapshot 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <styled.div>
        1
      </styled.div>
    </styled.div>
    <styled.div
      id={NaN}
    >
      Accordion Title
    </styled.div>
    <styled.div>
      <styled.div />
    </styled.div>
  </styled.div>
  <styled.div
    collapsed={true}
  >
     
     
  </styled.div>
</styled.div>
`;
